wordpress-mobile/WordPress-Android
# Feature/rtl audit login

Partially fixes #2278 - Fixes RTL layouts on sign in/up screens.

Some Views are not properly positioned in the RTL language mode. 

To test: 
1. Check layouts of the 
WordPress.com login (email) + email password screens
Selft-hosted site login (site URL) + username and password screens
Create Wordpress.com account screen
2. Change your locale to a RTL language (eg. Hebrew) or force RTL layouts in the developer options.
3. Check the layouts, see step 1

Doesn't fix the ViewPager swipe direction in the RTL mode on the first app screen.
